### PR TITLE
Admin can create categories

### PR DESCRIPTION
**Pull request by bryceholcomb**

### CODE REVIEW

File: app/views/items/show.html.erb [hunk 1]
```diff
@@ -1,5 +1,11 @@
 <div class="item">
-  <h2><%= @item.title %></h2>
-  <h3><%= number_to_currency @item.currency %></h3>
-  <p><%= @item.description %></p>
+  <h2>Title: <%= @item.title %></h2>
+  <h3>Price: <%= number_to_currency @item.currency %></h3>
```
Comment (kpearson) [inline app/views/items/show.html.erb:3]: Is there a test for this?

File: app/views/categories/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="category">
+  <h2>Name: <%= @category.name %></h2>
+  <% if @category.items %>
+    <h3>Items:</h3>
+    <% @category.items.each do |item| %>
```
Comment (kpearson) [inline app/views/categories/show.html.erb:5]: Maybe a category_items method.

File: app/views/items/show.html.erb (hunk 1 above)
Comment (bryceholcomb) [inline app/views/items/show.html.erb:3]: Yes for the view there is, I will add a test in the model spec for that specific method

File: app/views/categories/show.html.erb (hunk 2 above)
Comment (bryceholcomb) [inline app/views/categories/show.html.erb:5]: I'll start to add some helpers for things like this, also id like to make some of these partials after we build the functionality

**Pull request merged**